.github/workflows/ci.yml

```diff
@@ -6,8 +6,8 @@ on:
       - master
   pull_request:
   schedule:
-    # Run at 18:00 UTC every day
-    - cron: '0 18 * * *'
+    # Run multiple times a day as the successfull cached links are not checked every time.
+    - cron: '0 */3 * * *'
```

Comment:
I think you wrote the cron schedule wrong. This will run it every 3rd hour, i.e. 8x per day. https://crontab.guru/#0_*/3_*_*_*

Reply:
Yeah, sorry ,fixed.